composer.json

```diff
@@ -13,12 +13,13 @@
         "php": ">=5.5",
         "arbiter/arbiter": "^0.1",
         "destrukt/destrukt": "^0.7.1",
+        "equip/adr": "^1.0",
         "filp/whoops": "^1.1",
         "nikic/fast-route": "^0.7",
+        "monolog/monolog": "^1.17",
```

Comment:
Nah we don't have to change anything there. We do a similar thing with Plates already.